addons/l10n_mx/models/res_config_settings.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    l10n_mx_account_income_return_discount_id = fields.Many2one(
+        comodel_name="account.account",
+        string='Income Returns and Discounts Account',
+        readonly=False,
+        related='company_id.l10n_mx_income_return_discount_account_id',
+        domain="[('account_type', 'in', ('income', 'expense'))]",
+    )
```

Comment:
I think you need 2 separated accounts here, one for income, one for expense :thinking: 
The name of the field and the domain are confusing otherwise